src/lazy/binary/raw/v1_1/type_descriptor.rs

```diff
@@ -48,6 +62,7 @@ impl Opcode {
             (0xE, 0x0) => (IonVersionMarker, low_nibble, None),
             (0xE, 0x1..=0x3) => (SymbolAddress, low_nibble, Some(IonType::Symbol)),
             (0xE, 0xA) => (NullNull, low_nibble, Some(IonType::Null)),
+            (0xE, 0xB) => (TypedNull, low_nibble, Some(IonType::Null)),
```

Comment:
This is a little more involved. Right now all of the data needed for the `Header` is expected to be available from the first byte. So, any headers created by opcodes without an IonType would cause a panic.

This does make sense, it would just probably be better as a follow-up to allow types to be deferred. I'll create an issue for it.